models/field_definition_blackbox_test.go

```diff
@@ -1,18 +1,23 @@
-package models
+package models_test
 
 import (
 	"encoding/json"
 	"fmt"
 	"reflect"
 	"testing"
+
+	. "github.com/almighty/almighty-core/models"
+	"github.com/almighty/almighty-core/resource"
 )
 
 func TestListFieldDefMarshalling(t *testing.T) {
+	t.Parallel()
+	resource.Require(t, resource.UnitTest)
```

Comment:
I've given some thought to the resource concept. I think we would be better off if we'd call it "precondition". Then `resource.Database` would become `precondition.HAVE_DATABASE` and `resource.UnitTest` would become `precondition.RunUnitTest`. The `resource.Require()` function  could be renamed to `precondition.IsFulfilled()`.

In total this means:

```go
precondition.IsFulfilled(t, precondition.RunUnitTest)
```

Does this sound better @tsmaeder ? Feel free to create a PR an reference this discussion (https://github.com/almighty/almighty-core/pull/240#discussion_r80227219).